src/main/java/core/basesyntax/ReadWriteList.java

```diff
@@ -10,16 +11,25 @@ public class ReadWriteList<E> {
     private ReadWriteLock lock = new ReentrantReadWriteLock();
 
     public void add(E element) {
-        // write your code here
+        list.add(element);
```

Comment:
The element is added to the list before acquiring the write lock. This could lead to concurrency issues. You should acquire the write lock before modifying the list.